skbio/io/format/gff3.py

```diff
@@ -0,0 +1,469 @@
+file. GFF stands for Generic Feature Format. GFF files are plain
+text, 9-column, tab-delimited files [1]_.
+
+Format Support
+--------------
+**Has Sniffer: Yes**
+
++------+------+---------------------------------------------------------------+
+|Reader|Writer|                          Object Class                         |
++======+======+===============================================================+
+|Yes   |Yes   |:mod:`skbio.metadata.IntervalMetadata`                         |
++------+------+---------------------------------------------------------------+
+|Yes   |Yes   |:mod:`skbio.sequence.DNA`                                      |
++------+------+---------------------------------------------------------------+
+|Yes   |Yes   |:mod:`skbio.sequence.Sequence`                                 |
++------+------+---------------------------------------------------------------+
+|Yes   |Yes   |generator of :mod:`skbio.metadata.IntervalMetadata`            |
++------+------+---------------------------------------------------------------+
+
+Format Specification
+--------------------
+The first line of the file is a comment that identifies the format and
+version.  This is followed by a series of data lines, each one of
+which corresponds to an annotation.  The 9 columns of the annotation
+section are as follows:
+
++-------+------------------------------------------+
+|Column | Description                              |
++=======+==========================================+
+|SEQID  | ID of the landmark used                  |
++-------+------------------------------------------+
+|SOURCE | algorithm used to generate this feature  |
++-------+------------------------------------------+
+|TYPE   | type of the feature                      |
++-------+------------------------------------------+
+|START  | start of the feature                     |
++-------+------------------------------------------+
+|END    | end of the feature                       |
++-------+------------------------------------------+
+|SCORE  | floating point score                     |
++-------+------------------------------------------+
+|STRAND | The strand of the feature (+/-/./?)      |
++-------+------------------------------------------+
+|PHASE  | only for TYPE="CDS"                      |
++-------+------------------------------------------+
+|ATTR   | feature attributes                       |
++-------+------------------------------------------+
+
+
+Column 9 (ATTR) is list of feature attributes in the format
+tag=value. Multiple tag=value pairs are separated by
+semicolons. Multiple values of the same tag are indicated by
+separating the values with the comma ",". The following tags have
+predefined meanings:
+
+* ID. Indicates the unique identifier of the feature. IDs must be
+  unique within the scope of the GFF file.
+
+* Name. Display name for the feature. This is the name to be displayed
```

Comment:
The thing I copied directly in genbank is among line 118-130 and some examples in those lines. Is this a problem? I can try to rephrase it.